test/integration/JsonSerialization.test.ts

```diff
@@ -147,4 +148,10 @@
   }
 }`);
   });
+
+  it("should initialize resource type by default", () => {
+    const valueSet: ValueSet = new ValueSet();
+    expect(valueSet.resourceType).toEqual("ValueSet");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=6104340)